src/util/logging.cpp

```diff
@@ -87,37 +88,45 @@ void MessageHandler(QtMsgType type,
         Logfile.open(QIODevice::WriteOnly | QIODevice::Text);
     }
 
+    debugLevel = CmdlineArgs::Instance().getDebugLevel(); // Get verbosity level
+
     switch (type) {
-    case QtDebugMsg:
-        fprintf(stderr, "Debug %s", ba.constData());
+    // debugLevel is set to 1 by default, can be changed with the cmdline
```

Comment:
As I already mentioned, this intention can be captured by defining a constant for the default value.